src/modules/messages/index.ts

```diff
@@ -116,6 +116,19 @@ export class Messages {
     return response.data.messages as MessagesInterface[];
   }
 
+  public async searchMessages(
```

Comment:
i think creating an extra method is not necessary; it just needs to be documented that you cannot send a search along with an `orderBy` and a `whereCondition`. Creating an extra method would be redundant; you can use `getMessages`